packages/wrangler/scripts/bundle.ts

```diff
@@ -25,6 +25,18 @@ async function run() {
       "import.meta.url": "import_meta_url",
     },
   });
+
+  // custom miniflare cli
+  await build({
+    entryPoints: ["./src/miniflare-cli/index.ts"],
+    bundle: true,
+    outfile: "./miniflare-dist/index.mjs",
```

Comment:
you should add `miniflare-dist` to `package.json:files`

Reply:
Oh yes!!

Reply:
Because the wrangler package.json does not set the package level format to ESM, then a file with `.js` extension is assumed, by node, to be CommonJS format.
We could as esbuild to bundle this file in CommonJS format, but the ESM format is much cleaner, so I was trying to be forward looking - with a longer term goal of moving all the distributable code in this repo over to ESM.

Reply:
Oh the other reason!
Is that this file is using a top-level await - which is very nice. And that is only allowed in ESM formatted files, AFAIK.

Reply:
Awesome TIL